### ISSUE TITLE
No support for K55?

### ISSUE THREAD

**Issue by Darklink2050** [DarthAffe/KeyboardAudioVisualizer#58]
I have a Corsair K55 RGB and this program does not seem to be working

**Comment by DarthAffe** [comment]
Yes, sadly Corsair doesn't support the K55 inside the SDK. I don't know what's the reason for that, but that was already discussed in the corsair-forums quite some time ago.
I can't remember 100% sure but I think it had something to do with the K55 only having limited rgb-support (no per key-lighting)
